packages/shared/src/table/selectable-table.tsx

```diff
@@ -152,7 +156,7 @@ export const SelectableTable: SelectableTableProps = <
                         onSelect: onSelect,
                         isSelected: isRowSelected(getUID(row), selectedRows),
                         isDisabled:
-                          !isRowSelectable?.(row) ||
+                          (!!isRowSelectable && !isRowSelectable?.(row)) ||
```

Comment:
this is not needed... `!isRowSelectable?.(row)` syntax will only execute when function exists, else not...
```suggestion
                          (!isRowSelectable?.(row) ||
```

Reply:
original line was correct...

Reply:
The issue in original condition is, 
  isRowSelectable?: IsRowSelectable; is mentioned as optional. But if the component is not passing isRowSelectable, then !isRowSelectable?.(row)always returns True, and all the rows are disabled by default. It's not correct.

Reply:
got it !!
then probably we can remove `?.` part entirely... it's not needed when we are using condition like `!!isRowSelectable && ...`

Reply:
ack, is it won't affect fusion UI? i think fusion is using this table

Reply:
it should not affect in this case...

Reply:
I meant using `!!isRowSelectable && !isRowSelectable(row))` instead of `!!isRowSelectable && !isRowSelectable?.(row))`.

Reply:
`?.`, when we optionally call `isRowSelectable`.

Reply:
ack, changes done